README.md

```diff
@@ -2408,6 +2409,33 @@ karate.configure('ssl', { trustAll: true });
 
 For end-to-end examples in the Karate demos, look at the files in [this folder](karate-demo/src/test/java/ssl).
 
+### NTLM Authentication
+Karate provides support for NTLM authentication using the Apache NTLMEngine implementation.
+
+| Key           | Type   | Required? | Description                                                    |
+|---------------|--------|-----------|----------------------------------------------------------------|
+| `username`    | string | required  | NTLM username                                                  |
+| `password`    | string | required  | NTLM password                                                  |
+| `workstation` | string | optional  | The workstation the authentication request is originating from |
+| `domain`      | string | optional  | The domain to authenticate within                              |
+
+Example:
+```cucumber
+# enable NTLM authentication for the remaining scenario requests
```

Comment:
this is brilliant. may I request only one change, can you shorten the configure key to `ntlmAuth`

Reply:
thanks. I updated the PR with the shorter configure key of `ntlmAuth`.